__tests__/app.test.js

```diff
@@ -84,8 +84,61 @@ describe("GET /api/articles", () => {
                     comment_count: expect.any(String),
                 })
                 expect(article).not.toHaveProperty('body')
-                expect(articlesArray).toBeSorted()
+                expect(articlesArray).toBeSortedBy("created_at", { descending: true})
             })
         })
     })
 })
+
+
+describe("GET /api/articles/:article_id/comments", () => {
+    test("status 200: responds with array of all comments with the correct properties", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((res) => {
+            const commentsArray = res.body.comments
+            expect(commentsArray).toHaveLength(11)
+            commentsArray.forEach((comment) => {
+                expect(comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    article_id: expect.any(Number),
+                })
+                expect(commentsArray).toBeSortedBy("created_at", { descending: true})
+            })
+        })
+            })
+        })
+
+
+    describe("ERRORS - GET /api/articles/:article_id/comments", () => {
+        test("GET: 200 - returns an array if passed a valid article_id that has no associated comments", () => {
+        return request(app)
+        .get("/api/articles/2/comments")
+        .expect(200)
+        .then(({ body }) => {
+            console.log(body)
+            expect(body.comments).toEqual([])
+        })
+        })
```

Comment:
Great test but I wouldn't have it in the describe block for an error since it's a 200